src/utils/barGraph.ts

```diff
@@ -20,32 +21,23 @@ export const drawBarGraph = (
     const barPadding = 0.2;
 
     // x axis
-    const xScale = scaleBand()
-        .domain(categories)
-        .range([margin.left, graphWidth])
-        .padding(barPadding);
+    const xScale = scaleBand().domain(categories).range([margin.left, graphWidth]).padding(barPadding);
```

Comment:
Weird, `// prettier-ignore` should have stopped this from happening. I'm guessing you might not be using the correct formatter in vscode itself?